src/components/Accordion/__tests__/Accordion.spec.js

```diff
@@ -175,7 +175,7 @@ describe('<Accordion />', () => {
   });
 
   it('takes styled prop and wraps Accordion in slds-card', () => {
-    const mounted = render(
+    const mounted = mount(
```

Comment:
Why exactly did we render here in the first place? 🙈

Reply:
I dunno 😄

Reply:
It was my first Component :smiley: